packages/create/src/cli/convert/preview.ts

```diff
@@ -0,0 +1,47 @@
+import type { Command } from 'commander';
+import { getLogger, setLogLevelVerbose, traceChanges } from '../../tracing';
+import { convertToVirtualPreview } from '@sap-ux/app-config-writer';
+/**
+ * Add a new sub-command to convert the preview of a project to virtual files.
+ *
+ * @param {Command} cmd - The command to add the convert sub-command to.
+ */
+export function addConvertPreviewCommand(cmd: Command): void {
+    cmd.command('preview-config [path]')
+        .option('-s, --simulate', 'simulate only do not write or install')
```

Comment:
```suggestion
        .option('-s, --simulate', 'simulate only do not write or install')
```
Can the second string be sentence case or is it code?